### ISSUE TITLE
Support for live capture

### ISSUE THREAD

**Issue by alberto-bc** [brimsec/brim#1162]
It is possible to redirect packet data and feed it to [WireShark through pipes](https://wiki.wireshark.org/CaptureSetup/Pipes). This is useful when inspecting network data from another machine in real time.

Is this something that can be achieved in Brim as well? If not, are there any plans to support this scenario?

**Comment by philrz** [comment]
@alberto-bc: Thanks for your interest in Brim! FYI, it looks like that Wireshark page may be deprecated in favor of https://gitlab.com/wireshark/wireshark/-/wikis/CaptureSetup/Pipes.

I wasn't previously familiar with this technique, so I was just hacking around trying it out myself. I was unsuccessful, but am happy to speak with our Dev team to see if it can be made to work or, if not, scope how difficult it would be to add. Just in case I find the supportability varies per platform, are you running Brim on Windows, macOS, or Linux?

**Comment by alberto-bc** [comment]
> Just in case I find the supportability varies per platform, are you running Brim on Windows, macOS, or Linux?
Windows and macOS.

> Also, is there a use case you have in mind that specifically requires the use of pipes?
Our use case is to inspect network data from another machine in real time. This is achieved in Wireshark with pipes, but you are correct, at the end it's in the interest of ingesting live data.

**Comment by philrz** [comment]
@alberto-bc: Thanks for the clarity. I did speak with our Dev team and confirmed that this is unlikely to be a priority for them in the near term. Of course, that could change and/or someone from the community might volunteer to start working on it and/or something causes the priority to shift. Therefore, I'm going to close this one as a duplicate of the existing "live" issue #569 so we can continue to track accumulate interest in one place. I've made a comment there to ensure that pipes are considered as an incoming source if/when this work is picked up.
